x-pack/plugins/alerting/server/task_runner/rule_action_helper.test.ts

```diff
@@ -314,6 +334,16 @@ describe('rule_action_helper', () => {
     });
   });
 
+  // Christos, I need to understand something here
```

Comment:
Me too 😄 . What should be the `interval` for system actions?

Reply:
We decided to remove it for now since they no use for it in system actions